tests/cases/UboDeclarationsTest.php

```diff
@@ -78,6 +79,15 @@ function test_CreateUbo()
         $this->assertEquals($ubo->Birthplace, $newUbo->Birthplace);
     }
 
+    function test_throw_CreateUbo(){
+        $declaration = $this->getMatrixUboDeclaration();
+
+        if(empty($declaration->Id)){
```

Comment:
Why are you checking if this is empty? We need here a clear test that is trying to create an UBO without an ID and that is expecting an exception to be thrown.